### DISCUSSION TITLE
iCloud storage

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#971] [General]
What are the ways to get more iCloud storage?

**Comment by hernessteve** [comment]
Just buy some more storage and the issue is solved

**Comment by MirkosMangle** [comment]
That's not the only way. You can also have some free storage if you know what you are doing. Here is great article about it - [https://applenapps.com/get-more-icloud-storage/](url). First of all i would clean up all the duplicate photos on the iphone (believe me we all have a lot of those) and all the other unnecessary data which we store on our phones.
